### DISCUSSION TITLE
Make DefaultSceneProcessor variables protected

### DISCUSSION THREAD

**Discussion by JonasSchnettker** [FirstGearGames/FishNet#220] [Feature Request]
Since the documentation recommends inheriting from the DefaultSceneProcessor (https://fish-networking.gitbook.io/docs/manual/guides/scene-manager/addressables) and overriding its functions, its variables should be protected as well, as they are most likely to be used as well.

**Comment by FirstGearGames** [comment]
I can take a look at what variables would make sense at being protected. I'm marking this as accepted and will let you know what I find.

**Comment by FirstGearGames** [comment]
Added in 3.3.0
